app/routes/explore.js

```diff
@@ -0,0 +1,19 @@
+import AdminRoute from 'ghost-admin/routes/admin';
+import {inject as service} from '@ember/service';
+
+export default class ExploreRoute extends AdminRoute {
+    @service session;
+    @service store;
+    @service feature;
+
+    beforeModel() {
+        super.beforeModel(...arguments);
+        if ((!this.session.user.isAdminOnly && !this.session.user.isOwnerOnly) || !this.feature.explore) {
```

Comment:
`(!this.session.user.isAdminOnly && !this.session.user.isOwnerOnly)` isn't needed here because that's already handled by extending from `AdminRoute` 🙂

Reply:
Removed!